File: Sis104Avance/app/src/main/java/com/example/sis104avance/graficos/Grafico2dActivity.java
package com.example.sis104avance.graficos;

import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.example.sis104avance.R;

public class Grafico2dActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(new SierpinskiCarpet(Grafico2dActivity.this));


    }
}

File: trabajoSIS104/app/src/main/java/com/example/trabajosis104/ejercicios/Grafico2dActivity.kt
package com.example.trabajosis104.ejercicios

import android.os.Bundle
import androidx.activity.viewModels
import androidx.appcompat.app.AppCompatActivity

class Grafico2dActivity : AppCompatActivity() {

    private val graficoViewModel: GraficoViewModel by viewModels()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(Grafico2d(this))
    }
}
